.readthedocs.yml

```diff
@@ -19,14 +19,15 @@ sphinx:
 # Optionally build your docs in additional formats such as PDF and ePub
 formats: all
 
-# Optionally set the version of Python and requirements required to build your docs
+# Optionally set the Python requirements required to build your docs
 python:
-  version: 3.6
   install:
     - requirements: docs/requirements.txt
 
 #conda:
 #  environment: environment-docs.yml
 
 build:
-  image: stable
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.6"
```

Comment:
Unless something breaks during build, 3.8 at least, since this is already EOL for a while.
(could also be a separate PR)

Reply:
Version updated to 3.10, readthedocs seems to build successfully.

Reply:
Very weird that even latest commit built RTD without problem with the options that you found were causing errors: 
https://github.com/bird-house/birdhouse-deploy/commit/ce5c14df65ebc0dbfc50e097dee0d2b50a0007f5

But your latest one is also good:
[`90611a7` (#360)](https://github.com/bird-house/birdhouse-deploy/pull/360/commits/90611a7984e5e1ef59c3562d77818e69af55e40f)

Funky stuff happening recently on RTD it seems...

Reply:
The reason the latest build on master could be successful was because RTD is deprecating the option progressively, doing temporary deprecation enforcements until in a few weeks where the option becomes fully deprecated (see [blog here](https://blog.readthedocs.com/use-build-os-config/)). I just happened to build it on Sept. 18th where I got the deprecation error.